tokens/global/chip.json5

```diff
@@ -1,26 +1,25 @@
 {
   color: {
-    docCategory: 'colors',
```

Comment:
If we want to keep the reference to `docCategory` and `docExample` within the same file we can change the names to something like `docCategoryChip` and `docExampleChip` and that should remove the collisions